src/web/components/ArticleBody.tsx

```diff
@@ -38,7 +38,7 @@ const bodyStyle = (display: Display) => css`
 `;
 
 const linkColour = (palette: Palette) => css`
-	a {
+	a:not([data-ignore-global-link-styling]) {
```

Comment:
Is it worth following a key-valuation convention (like print.css https://github.com/guardian/dotcom-rendering/blob/main/src/static/css/print.css#L1)? I don't mind this, but I wonder if you can do `data-ignore="global-link-style"` so if you need to add `data-ignore="global-list-style` it follows the convention? Otherwise the value of `true` in the data attribute is not really meaningful (it could be `false` with the same effect)